utilities/transactions/write_prepared_txn_db.cc

```diff
@@ -505,6 +517,12 @@ void WritePreparedTxnDB::AddCommitted(uint64_t prepare_seq, uint64_t commit_seq,
         // legit when a commit entry in a write batch overwrite the previous one
         max_evicted_seq = evicted.commit_seq;
       }
+#ifdef OS_LINUX
```

Comment:
Not entirely sure how weak symbol works on MacOS, but I got a linking error of undefined reference on CircleCI OSX and MS Windows Visual studio. Some online search suggest that people have similar issues, but I didn't investigate too much because I thought most of this stress tests run on Linux. Let me give it one more shot.

Reply:
If we do not do `ifndef OS_LINUX` here, then we need to do it for the declaration of the function. If we want to get rid of the linking error for OS X and MSVC, we need to use two other tricks (https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/WeakLinking.html and https://devblogs.microsoft.com/oldnewthing/20200731-00/?p=104024). Not sure if it's better than current approach.

Reply:
OK